viewer/static/js/app/components/containers/app-container.js

```diff
@@ -147,7 +154,7 @@ class AppContainer extends React.Component {
       populationIds: this.populationIds,
 
       metricIdsToShow: this.metricIdsToShow,
-      allMetricIds: this.allMetricIds,
+      allMetricIdsInDataset: this.allMetricIdsInDataset,
```

Comment:
Hmm... technically I suppose it's available via `currentDataset.metrics`. I'll look into that.